sample_code/ep053/rev05/config.py

```diff
@@ -0,0 +1,2 @@
+SECRET_KEY = 'secret_random_string'
+WTF_CSRF_SECRET_KEY = 'secret_random_string2'
```

Comment:
Fun fact: if you use only the `WTF_CSRF_SECRET_KEY`, Flask crashes (500 error), because Flask itself demands `SECRET_KEY` to be set. Weird setup, but anyways 🍡

Reply:
makes sense -- they're used for two different thing (one for sessions, the other for csrf tokens)